packages/SwingSet/misc-tools/audit-refgraph.py

```diff
@@ -0,0 +1,115 @@
+import sys, json, re
+from collections import defaultdict
+
+# first build an underscore-separated list of all kvStore values
+#  sqlite3 -separator _ ss.sqlite 'SELECT * FROM kvStore' |sort >all-kv.txt
+
+# then feed that into stdin
```

Comment:
...to accomplish what?

Reply:
good point, this is to support #8759 but really wants to be rewritten in JS and using direct DB access, it's more of a quick sketch. @siarhei-agoric since #8759 is assigned to you, when you implement it, remove this python script and replace it with whatever you create. But for now it's still useful.

I'll add some notes.